Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The HamburgerMenuItem provides an abstract implementation for HamburgerMenu entries.
+    /// </summary>
+    public abstract class HamburgerMenuItem
+    {
+        /// <summary>
+        /// Gets or sets gets of sets a value that specifies label to display.
+        /// </summary>
+        public string Label { get; set; }
```

Comment:
Shouldn't these properties be dependency properties? Means no-one can bind to these otherwise.